### PR TITLE
feat(cisco_wsa): Add support for Splunk recommended w3 format logs

### PR DESCRIPTION
**Pull request by darshm-crest**

### CODE REVIEW

**Comment by nandinivij** [comment]
Hi @darshm-crest ,
Currently, in wsa logs we have source type-cisco:wsa:w3c but to SC4S filter we are adding [cicso:wsa:w3c:recommended]. If Filter is released now before a new addon with this source type is released, will it be an issue for current addon users?  https://docs.splunk.com/Documentation/AddOns/released/CiscoWSA/Datatypes

**Review by nandinivij — APPROVED** [review]

**Review by rfaircloth-splunk — APPROVED** [review]

File: docs/sources/Cisco/index.md [hunk 1]
```diff
@@ -522,6 +523,7 @@ Verify timestamp, and host values match as expected
 | cisco_wsa    | cisco:wsa:l4tm    | netproxy          | None     |
 | cisco_wsa    | cisco:wsa:squid    | netproxy          | None     |
 | cisco_wsa    | cisco:wsa:squid:new    | netproxy          | None     |
+| cisco_wsa    | cisco:wsa:w3c:recommended    | netproxy          | None     |
```
[the remark this answers is not in the document]
Comment (darshm-crest) [reply docs/sources/Cisco/index.md:526]: As per our discussion with Mark, we have kept the sourcetype name as cisco:wsa:w3c:recommended.

**Pull request merged**